jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -246,6 +246,25 @@ ecma_op_general_object_get_property (ecma_object_t *obj_p, /**< the object */
 } /* ecma_op_general_object_get_property */
 
 /**
+ * Checks whether the property name is "length".
+ *
+ * @return true if the property name is length
+ *         false otherwise
+ */
+static bool
+ecma_op_general_object_property_name_is_length (ecma_string_t *property_name_p) /**< property name */
```

Comment:
Is this function inlined? Since it is used in release code only once, I'd suggest explicitly asking the compiler to inline this. (`inline` and/or `__attr_always_inline___`)